jnosql-metamodel-extension/src/main/java/org/eclipse/jnosql/mapping/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Copyright (c) 2022 Otávio Santana and others
+ *   All rights reserved. This program and the accompanying materials
+ *   are made available under the terms of the Eclipse Public License v1.0
+ *   and Apache License v2.0 which accompanies this distribution.
+ *   The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ *   and the Apache License v2.0 is available at http://www.opensource.org/licenses/apache2.0.php.
+ *
+ *   You may elect to redistribute this code under either of these licenses.
+ *
+ *   Contributors:
+ *
+ *   Alessandro Moscatelli
+ */
+
+/**
+ * MongoDB is a source-available cross-platform document-oriented database program.
```

Comment:
This one does not belong only to MongoDB anymore.
Please, don't forget to update the package info here.

Reply:
Should be ok now !